src/bsp/src/stm32/src/interloc/src/InterlocTimeManager.cpp

```diff
@@ -0,0 +1,75 @@
+#include "interloc/InterlocTimeManager.h"
+#include <interloc/Decawave.h>
+
+InterlocTimeManager::InterlocTimeManager(IBSP& bsp) : m_bsp(bsp), m_numSlots(1), m_slotId(1) {
+    updateTimings();
+}
+
+void InterlocTimeManager::setNumSlots(uint16_t numSlots) {
+    m_numSlots = numSlots;
+    updateTimings();
+}
+
+void InterlocTimeManager::setSlodId(uint16_t slotId) {
+    m_slotId = slotId;
+    updateTimings();
+}
+
+void InterlocTimeManager::updateTimings() {
+    volatile uint64_t pollRxToResponseTxOffset =
+        POLL_AIR_TIME_WITH_PREAMBLE_US + POLL_TO_FIRST_RESPONSE_GUARD_US +
+        (uint64_t)(m_slotId - 1) *
+            (uint64_t)(RESPONSE_AIR_TIME_WITH_PREAMBLE_US + RESPONSE_RX_TO_RESPONSE_TX_GUARD_US);
```

Comment:
I think it would be nice to add a marker of Host/Remote in the names where there is Rx and Tx.

In this example, who is in receive and who is in transmit, this instance of the code, or an other robot
maybe do : RESPONSE_hRX_TO_rRESPONSE_TX_GUARD_US to avoid having tooo long names

the comment applies to all constants

Reply:
Removed RX and TX from names instead